INSTALL.md

```diff
@@ -32,6 +34,36 @@ How to install
 
 See also [repology's cross-distro index](https://repology.org/metapackage/zsh-syntax-highlighting/versions)
 
+Second, enable zsh-syntax-highlighting by sourcing the script. Running this command on the terminal will add the source line to the end of your .zshrc:
+
+* On most Linux distributions (except perhaps NixOS):
+
+```zsh
```

Comment:
Please indent everything from the ```` ```zsh ```` to the ```` ``` ```` four spaces, as done in the numbered lists later in the document.  That will cause the preformatted block to be indented under the bullet, rather than at the same level as it.